content/security/advisory/2017-08-08.adoc

```diff
@@ -0,0 +1,31 @@
+---
+layout: simplepage
+title: Jenkins Security Advisory 2017-08-08
+section: security
+kind: plugin
+---
+
+This advisory announces a vulnerability in the plugin:saml[SAML Plugin].
+
+== Description
+
+=== SAML Plugin stored keystore and private key passwords in plain text
+*JENKINS-47003*
```

Comment:
Huh? Should be [JENKINS-46007](https://issues.jenkins-ci.org/browse/JENKINS-46007). What is 47003?